incubator/binding-pgsql/src/main/moditect/module-info.java

```diff
@@ -15,6 +15,11 @@
 module io.aklivity.zilla.runtime.binding.pgsql
 {
     requires io.aklivity.zilla.runtime.engine;
+    requires org.antlr.antlr4.runtime;
+
+    exports io.aklivity.zilla.runtime.binding.pgsql.parser;
+    exports io.aklivity.zilla.runtime.binding.pgsql.parser.listener;
+    exports io.aklivity.zilla.runtime.binding.pgsql.parser.module;
```

Comment:
```suggestion
    exports io.aklivity.zilla.runtime.binding.pgsql.parser.model;
```
Perhaps you meant `model`?

Reply:
yeah thanks!